src/components/CStatusCard/index.tsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 
 import capitalizeFirstLetter from '@/utils/capitalizeFirstLetter';
 
-type status = 'active' | 'completed' | 'cancelled';
+export type StatusType = 'active' | 'completed' | 'expired' | 'pending';
```

Comment:
There's no 'active' type in the back-end code, and it's the same as `pending`, so I suggest removing it.